File: Components/DevExpressVCL/Demos/ExpressPDFViewer/Delphi/Viewer/uExportToBitmaps.pas
unit uExportToBitmaps;

{$I cxVer.inc}

interface

uses
{$IFDEF DELPHI16}
  System.UITypes,
{$ENDIF}
  cxGraphics, cxControls, cxLookAndFeels, cxLookAndFeelPainters, cxContainer, cxEdit,
  dxLayoutcxEditAdapters, dxLayoutControlAdapters, dxLayoutContainer, cxClasses, cxCheckBox, StdCtrls, cxTextEdit,
  cxMaskEdit, cxSpinEdit, Classes, Controls, dxLayoutControl, uExportToFileDialog;

type
  { TfrmExportToBitmaps }

  TfrmExportToBitmaps = class(TfrmExportToFileDialog)
    teFilePrefix: TcxTextEdit;
    dxLayoutItem2: TdxLayoutItem;
    dxLayoutAutoCreatedGroup3: TdxLayoutAutoCreatedGroup;
    procedure FormCreate(Sender: TObject);

  end;

implementation

{$R *.dfm}

procedure TfrmExportToBitmaps.FormCreate(Sender: TObject);
begin
  inherited;
  cbOpenAfterExport.Caption := 'Open folder after export';
end;

end.

File: Components/DevExpressVCL/Demos/ExpressPDFViewer/CBuilder/Viewer/uExportToBitmaps.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "uExportToBitmaps.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "cxCheckBox"
#pragma link "cxClasses"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxEdit"
#pragma link "cxGraphics"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxLookAndFeels"
#pragma link "cxMaskEdit"
#pragma link "cxSpinEdit"
#pragma link "cxTextEdit"
#pragma link "dxLayoutContainer"
#pragma link "dxLayoutControl"
#pragma link "dxLayoutControlAdapters"
#pragma link "dxLayoutcxEditAdapters"
#pragma resource "*.dfm"
TfrmExportToBitmaps *frmExportToBitmaps;
//---------------------------------------------------------------------------
__fastcall TfrmExportToBitmaps::TfrmExportToBitmaps(TComponent* Owner)
	: TfrmExportToFileDialog(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TfrmExportToBitmaps::FormCreate(TObject *Sender)
{
  cbOpenAfterExport->Caption = "Open folder after export";
}
//---------------------------------------------------------------------------

